lib/utils.js

```diff
@@ -33,7 +33,8 @@ function debrack (s) {
 }
 
 function uriToFilename (uri, base) {
-  var filename = path.join(base, uri)
+  var decoded = uri.split('/').map(decodeURIComponent).join('/')
```

Comment:
Maybe add   *  ?  \    
?

Or wait until the file system objects?

Example use case: I want to copy my entire Mac home folder into a solid server to sync it with other macs